curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparisons-with-the-logical-and-operator.md

```diff
@@ -33,7 +33,7 @@ return "No";
 
 # --instructions--
 
-Replace the two if statements with one statement, using the && operator, which will return `"Yes"` if `val` is less than or equal to `50` and greater than or equal to `25`. Otherwise, will return `"No"`.
+Replace the two if statements with one statement, using the `&&` operator, which will return `Yes` if `val` is less than or equal to `50` and greater than or equal to `25`. Otherwise, will return `No`.
```

Comment:
```suggestion
Replace the two if statements with one statement, using the `&&` operator, which will return the string `Yes` if `val` is less than or equal to `50` and greater than or equal to `25`. Otherwise, will return the string`No`.
```